docs/src/pages/docs/agents/01-agent-memory.mdx

```diff
@@ -350,12 +360,123 @@ await memory.deleteThread(thread.id);
 
 Note that in most cases, you won't need to manage threads manually since the agent's `generate()` and `stream()` methods handle thread management automatically. Manual thread management is primarily useful for advanced use cases or when you need more fine-grained control over the conversation history.
 
+### Working Memory
```

Comment:
I guess this file wasn't formatted before - sorry for all the formatting changes here, prettier auto formatted on save